bookwyrm_backend/src/main/java/com/bookwyrm/backend/book/dao/BookDao.java

```diff
@@ -16,7 +18,8 @@
     private String description;
     private String author;
     private String isbn;
-    private String genre;
```

Comment:
Same as above, genre is necessary

Reply:
My mistake. if that's the official genre for the book, I'll add it back.

Reply:
yes, thats the official genre